### PR TITLE
Update Docker Images to the recent versions of FuseSOC and tools

### PR DESCRIPTION
**Pull request by Nancy-Chauhan**
**This is Work in progress to solve the issue :** 

librecores/librecores-ci/issues/49.

**Work Left** : 

Update LibreCores Pipeline Library so that we run demos with recent versions of the tools.

### CODE REVIEW

File: librecores-ci-modules-fpga/Dockerfile [hunk 1]
```diff
@@ -46,7 +46,7 @@ RUN pip install pytest tap.py
 # Cocotb doesn't work well with a centralized installation because it
 # compiles files in the installation directory. Hence install it here.
 RUN git clone https://github.com/potentialventures/cocotb.git
-WORKDIR cocotb
+WORKDIR /cocotb
```
Comment (oleg-nenashev) [inline librecores-ci-modules-fpga/Dockerfile]: Env var below is still the same: "ENV COCOTB=/tmp/cocotb"

File: librecores-ci-modules/Dockerfile [hunk 2]
```diff
@@ -46,7 +46,7 @@ RUN pip install pytest tap.py
 # Cocotb doesn't work well with a centralized installation because it
 # compiles files in the installation directory. Hence install it here.
 RUN git clone https://github.com/potentialventures/cocotb.git
-WORKDIR cocotb
+WORKDIR /cocotb
```
Comment (oleg-nenashev) [inline librecores-ci-modules/Dockerfile]: same

**Review by oleg-nenashev — CHANGES_REQUESTED** [review]
Likely cocotb won't work after the change

**Comment by Nancy-Chauhan** [comment]
@oleg-nenashev Updated LibreCores CI Docker images to newer tool versions, updates images can be used further for the mor1kx project

**Review by oleg-nenashev — APPROVED** [review]
Has it been tested?

**Comment by Nancy-Chauhan** [comment]
@oleg-nenashev  Yes, I checked on a local machine.

**Comment by oleg-nenashev** [comment]
Update: it was not a change for mor1kx flow support

**Pull request merged**